lib/simple_google_auth.rb

```diff
@@ -33,7 +33,9 @@ def self.uri(state)
       response_type: "code",
       client_id:     config.get_or_call(:client_id),
       redirect_uri:  config.redirect_uri,
-      state:         state
+      access_type:   "offline",
+      state:         state,
+      approval_prompt: "force"
```

Comment:
There's a hook to allow individual apps to set additional parameters; I think these two are better set by that hook as not all users might want offline access.